.gitignore

```diff
@@ -261,8 +267,3 @@ airflow-build-dockerfile*
 *.zip
 
 _api/
-
-# Items generated when building the docs locally for release
-/docs/_build/
-/docs/_doctrees/*/
-/docs/_inventory_cache/
```

Comment:
@potiuk providers doc files are built in `docs/` path
removing it causes many untracked files.
![Screenshot 2025-04-07 at 10 48 27](https://github.com/user-attachments/assets/f2cfd1a2-32dd-4deb-96bc-98789f59cef2)